bolt/lib/Rewrite/RewriteInstance.cpp

```diff
@@ -82,6 +82,8 @@ extern cl::opt<bool> Hugify;
 extern cl::opt<bool> Instrument;
 extern cl::opt<JumpTableSupportLevel> JumpTables;
 extern cl::opt<bool> KeepNops;
+extern cl::opt<bool> Lite;
```

Comment:
Why was extern for Lite was added? Doesn't look like it's used.

Reply:
Lite is used in a few spots - lines 2069, 2075, 2081, etc